ignite/pkg/cosmosgen/cosmosgen.go

```diff
@@ -116,7 +120,8 @@ type generator struct {
 	sdkImport    string
 	deps         []gomodule.Version
 	appModules   []module.Module
-	thirdModules map[string][]module.Module // app dependency-modules pair.
+	appIncludes  []string
+	thirdModules map[string]ModuleIncludes // app dependency-modules/includes pair.
```

Comment:
Maybe it would make sense semantically:

```suggestion
	appIncludes  []string
	thirdModules map[string][]module.Module
	thirdModuleIncludes map[string][]string
```

It would allow to keep pre existing code unchanged and also to avoid writing `modules.Modules...`.

Reply:
Yep. Guess we could do it that way

Reply:
Implemented in #3696